src/pages/MainPage/components/AddTodo/AddTodo.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { pallete } from "../../../../misc/pallete";
+
+export const checkTodo = (todo: ITodoDetails) => {
+  let errors: string[] = [];
+  Object.keys(todo).forEach((key) => {
+    const value = todo[key as keyof ITodoDetails];
+    if (value === "" && key !== "description" && key !== "urgent") {
```

Comment:
jak działa to sprawdzenie? Brak wartości rozumiem, mam problem z dalszą częscią

Reply:
Funkcja sprawdza czy do inputów została wprowadzona jakaś wartość. Jako, że założyłem, że description jest optional to nie chciałbym, żeby funkcja brała ten klucz pod uwagę. Urgent zostało tam z jakichś wcześniejszych wersji i już się go pozbyłem :D